src/base/scanfoldersmodel.h

```diff
@@ -64,8 +64,8 @@ class ScanFoldersModel : public QAbstractListModel
 
     enum PathType
     {
-        DOWNLOAD_IN_WATCH_FOLDER,
         DEFAULT_LOCATION,
+        DOWNLOAD_IN_WATCH_FOLDER,
         CUSTOM_LOCATION
```

Comment:
Don't do this. Unless I am mistaken, we save these as numbers in the conf file. Older configurations will have their settings borked and this setting is quite important. It will mess their files.